stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -24,28 +24,49 @@ public struct PartialAsyncTask {
 public struct UnsafeContinuation<T> {
   private var context: UnsafeRawPointer
 
-  public func resume(_: __owned T) { }
+  /// Return a value into the continuation and make the task schedulable.
+  ///
+  /// The task will never run synchronously, even if the task does not
+  /// need to be resumed on a specific executor.
+  ///
+  /// This is appropriate when the caller is something "busy", like an event
+  /// loop, and doesn't want to be potentially delayed by arbitrary work.
+  public func resume(returning _: __owned T) { }
```

Comment:
TODO: rebase; we moved those in a previous PR over here, make sure it's all aligned.